### PR TITLE
Feat/popup comingsoon

### PR DESCRIPTION
**Pull request by PriscilliaAmmeux**
# Description
What are changes related to ? add popup for edit user (coming soon)

# Linked Issues
What are the issues fixed by this pull request ? #1103 

# Changes
add popup for edit user (coming soon)

# Screenshots
Put screenshots (if any)
image

# Tests
How has it been tested ?

- [X] Manually
- [ ] Automated tests
- [ ] QA
- [ ] Other

# Additional information
Precise any other information

### CODE REVIEW

File: yaki_admin/src/constants/modalMode.enum.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@
  */
 export enum MODALMODE {
   userDelete = "userDelete",
+  userEdit = "userEdit",
```
Comment (LaurentMag) [inline yaki_admin/src/constants/modalMode.enum.ts:14]: change userEdit with "comingSoon"

File: yaki_admin/src/ui/components/UserInfoCard.vue [hunk 2]
```diff
@@ -23,9 +21,15 @@ const props = defineProps({
...
   teammateStore.setIdOfTeammateToDelete(props.user.teammateId);
-  modalStore.setTeammateNameToDelete(`${props.user.firstname} ${props.user.lastname}`);
+  modalStore.setTeammateNameToDelete(
+    `${props.user.firstname} ${props.user.lastname}`
+  );
   modalStore.switchModalVisibility(true, MODALMODE.userDelete);
 };
+
+const OpenModalNotImplemented = () => {
+  modalStore.switchModalVisibility(true, MODALMODE.userEdit);
```
Comment (LaurentMag) [inline yaki_admin/src/ui/components/UserInfoCard.vue:31]: MODALMODE.comingSoon

File: yaki_admin/src/ui/views/ModalFrameView.vue [hunk 3]
```diff
@@ -46,11 +52,20 @@ const onAccept = async () => {
...
         @on-accept="onAccept"
         @on-cancel="onCancel"
-        v-else-if="modalStore.getMode === MODALMODE.teamDelete || modalStore.getMode === MODALMODE.userDelete" />
+        v-else-if="
+          modalStore.getMode === MODALMODE.teamDelete ||
+          modalStore.getMode === MODALMODE.userDelete
+        " />
+      <modal-coming-soon
+        @close="closeModal"
+        v-else-if="modalStore.getMode === MODALMODE.userEdit" />
```
Comment (LaurentMag) [inline yaki_admin/src/ui/views/ModalFrameView.vue:68]: MODALMODE.comingSoon

**Review by LaurentMag — CHANGES_REQUESTED** [review]

File: yaki_admin/src/ui/components/UserInfoCard.vue [hunk 4]
```diff
@@ -23,9 +21,15 @@ const props = defineProps({
 
 const UserToBeRemoved = () => {
   teammateStore.setIdOfTeammateToDelete(props.user.teammateId);
-  modalStore.setTeammateNameToDelete(`${props.user.firstname} ${props.user.lastname}`);
+  modalStore.setTeammateNameToDelete(
+    `${props.user.firstname} ${props.user.lastname}`
```
Comment (NyraSama) [inline yaki_admin/src/ui/components/UserInfoCard.vue:25]: Maybe create a fullname property to make it more clear because used in repetition

**Review by NyraSama — COMMENTED** [review]
Please consider comments

**Review by LaurentMag — APPROVED** [review]

**Pull request merged**